get-new-tool-versions/parsers/verify-added-to-image/verify-common-tool-parser.psm1

```diff
@@ -0,0 +1,25 @@
+function Search-ToolsVersionsNotOnImage {
+    param (
+        [string]$ToolName,
+        [string]$ReleasesUrl,
+        [string]$FilterParameter,
+        [string]$FilterArch
+    )
+    
+    $stableReleases = (Invoke-RestMethod $ReleasesUrl) | Where-Object stable -eq $true
+    if ($ToolName -eq "Node") {
+        $stableReleaseVersions = $stableReleases | ForEach-Object {$_.$FilterParameter.split(".")[0] + ".0"} |
+            Select-Object -Unique
+    } else {
+        $stableReleaseVersions = $stableReleases | ForEach-Object {$_.$FilterParameter.split(".")[0,1] -join"."} |
+            Select-Object -Unique
+    }
+    $toolsetUrl = "https://raw.githubusercontent.com/actions/virtual-environments/main/images/win/toolsets/toolset-2022.json"
+    $latestVersion = ((Invoke-RestMethod $toolsetUrl).toolcache |
```

Comment:
Let's replace $latestVersion -> $latestMinorVesion , remove () and use Foreach to $_.versions.Replace("*","0").

Reply:
Updated